flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NettyShuffleEnvironment.java

```diff
@@ -84,6 +87,9 @@
 	private static final String METRIC_OUTPUT_POOL_USAGE = "outPoolUsage";
 	private static final String METRIC_INPUT_QUEUE_LENGTH = "inputQueueLength";
 	private static final String METRIC_INPUT_POOL_USAGE = "inPoolUsage";
+	private static final String METRIC_INPUT_FLOATING_BUFFERS_USAGE = "floatingBuffersUsage";
+	private static final String METRIC_INPUT_EXCLUSIVE_BUFFERS_USAGE = "exclusiveBuffersUsage";
+
```

Comment:
remove this extra empty line

Reply:
ok，the checkstyle seems not cover this, so i overlooked this.

Reply:
it is not covered by checkstyle, but I think it should be common sense. :)